pkg/specgen/generate/container.go

```diff
@@ -136,8 +136,14 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 			return nil, fmt.Errorf("unable to process variables for --env-merge %s: %w", e, err)
 		}
 		splitWord := strings.Split(processedWord, "=")
+
 		if _, ok := defaultEnvs[splitWord[0]]; ok {
 			defaultEnvs[splitWord[0]] = splitWord[1]
+		} else if len(splitWord) > 1 {
```

Comment:
This len check must be above the if statement. This is important for both branches here. The current version can panic.
And in general it makes no sense to have two branches here. They do exactly the same, no?

Reply:
I agree, had it the other way round and then split it up because it _seemed_ clear demarcation of purposes here, but it doesn't make sense, thanks!